server/circleci.go

```diff
@@ -49,11 +49,17 @@ func (s *Server) triggerCircleCiIfNeeded(ctx context.Context, pr *model.PullRequ
 		return
 	}
 
+	blockList := s.Config.BlacklistPaths
```

Comment:
To clarify, does this repo level list act as an override to the global list? Or is the global list supposed to go away? If it's the latter, then it might create considerable bloat in the config because we would need to add a block list for every repo that is tracked.

I am assuming this acts only as an override.

Reply:
As requested by @jupenur the global list should be kept as there are certain files that are blocked in every repo. The repo specific list is an extension of the global list and the union of both is used.

Reply:
Right, thanks.